src/Notes/WC_Admin_Notes.php

```diff
@@ -164,7 +164,7 @@ public static function delete_all_notes() {
 				'orderby'    => 'date_created',
 				'per_page'   => 25,
 				'page'       => 1,
-				'type'       => array( 'info', 'marketing', 'warning' ),
+				'type'       => array( 'info', 'marketing', 'warning', 'survey' ),
```

Comment:
Could we use the consts from WC_Admin_Note?

Reply:
Sounds good to me! I changed it in the commit 3cf623e